openssl/src/x509/mod.rs

```diff
@@ -424,6 +424,22 @@ impl<'a> X509Ref<'a> {
         }
     }
 
+    /// Returns Issuer validity notAfter
+    pub fn not_after(&self) -> Asn1TimeRef {
```

Comment:
The macro's load directly from the internal structures, so that's not necessary. It probably should return an `Option` though, unless notBefore and notAfter are required to be present.